nbodykit/algorithms/paircount_tpcf/tpcf.py

```diff
@@ -188,6 +192,58 @@ def load(cls, output, comm=None):
         self.comm = comm
         return self
 
+    def to_xil(self, ells, mu_range=None, mu_sel=None, return_mu_sel=False):
+        r"""
+        Invert the measured wedges :math:`\xi(r,mu)` into correlation
+        multipoles, :math:`\xi_\ell(r)`.
+
+        Parameters
+        ----------
+        ells : array_like
+            the list of multipoles to compute
+        mu_range: array_like, optional
```

Comment:
This API seems to be a bit redunant with mu_sel. Is this absolutely necessary? Perhaps passing in range(mu_range[0], mu_range[1]) would work the same?

Reply:
Then you can probably also get rid of return_mu_sel because mu_sel is always an input.